plugins/inputs/sqlserver/sqlserverqueries.go

```diff
@@ -1170,25 +1170,24 @@ WITH utilization_cte AS
 ),
 processor_Info_cte AS
 (
-	SELECT (cpu_count / hyperthread_ratio) as number_of_physical_cpus
-	  FROM sys.dm_os_sys_info
+	SELECT ([cpu_count] / [hyperthread_ratio]) as [number_of_physical_cpus]
```

Comment:
Out of curiosity, what's the purpose for the square brackets?

Reply:
The square bracket does not change anything is just a way to encapsulate the column name, in SQL Server you can use `"col"` or `[col]`. it's mandatory and automatically used when the column name contains special chars or spaces, otherwise, it's not strictly needed.

I like to always use it, but it was also used in the whole query, except for that line